index.html

```diff
@@ -416,6 +417,12 @@ <h2><dfn>getEntriesByName()</dfn> method</h2>
           </section>
         </section>
       </section>
+      <section>
+        <h2><dfn>takeRecords()</dfn> method</h2>
+        <p>The <a>takeRecords()</a> method must return a copy of the
+          <a>context object</a>'s <a>observer buffer</a>, and also empty
```

Comment:
Let me know if this is being overly picky.

I'd prefer "and then empty" vs "and also empty", as "also" doesn't as strongly imply an order to these operations.